netlify.toml

```diff
@@ -4,7 +4,7 @@ publish = "site/public/"
 command = "pwd && cd themes/docsy && git submodule update -f --init && cd ../.. && hugo"
 
 [build.environment]
-HUGO_VERSION = "0.68.3"
+HUGO_VERSION = "0.74.3"
```

Comment:
any reason we dont do latest version ? how did we pick 0.74.3 ? 
https://github.com/gohugoio/hugo/releases

Reply:
I picked 0.74.3 because that's the lowest version with the util function I needed. But sure I've upgraded it to the latest `0.83.1` in ed7827913eadf498de154e5527907afe3a21d182 and didn't see any breaking changes. Thanks!